packages/clay-tabs/src/__tests__/index.tsx

```diff
@@ -0,0 +1,175 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import ClayTabs from '..';
+import React from 'react';
+import {cleanup, fireEvent, render} from '@testing-library/react';
+
+const ClayTabsWithItems = () => {
+	const [activeTabKeyValue, setActiveTabKeyValue] = React.useState<number>(0);
+
+	const onClick = (val: number) => setActiveTabKeyValue(val);
+
+	return (
+		<>
+			<ClayTabs>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 0}
+					onClick={onClick}
+					tabName="Dummy1"
+					tabkey={0}
+				/>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 1}
+					data-testid="tabItem2"
+					onClick={onClick}
+					tabName="Dummy2"
+					tabkey={1}
+				/>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 2}
+					onClick={onClick}
+					tabName="Dummy3"
+					tabkey={2}
+				/>
+			</ClayTabs>
+			<ClayTabs.Content activeTabKey={activeTabKeyValue}>
+				<ClayTabs.TabPane data-testid="tabPane1" tabkey={0}>
+					{'Tab Content 1'}
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane data-testid="tabPane2" tabkey={1}>
+					{'Tab Content 2'}
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane tabkey={2}>
+					{'Tab Content 3'}
+				</ClayTabs.TabPane>
+			</ClayTabs.Content>
+		</>
+	);
+};
+
+describe('ClayTabs', () => {
+	afterEach(cleanup);
+
+	it('renders', () => {
+		const {container} = render(<ClayTabs />);
+
+		expect(container).toMatchSnapshot();
+	});
```

Comment:
Don't we usually have blank lines between `it()` blocks? (Please ignore me if I'm mistaken about that)

Reply:
Yes, I guess that is a pattern here. 😅

Reply:
I gonna fix it. Would be awesome add an eslint rule for it

Reply:
AFAIK there is not en existing ESLint rule that we could use for this; the closest I could find was [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements), which has a zillion configuration possibilities but none which would apply here.

We might have to make a custom rule for it. In any case, created a ticket for it [https://github.com/liferay/eslint-config-liferay/issues/65](here).